app/models/concerns/bulkrax/import_behavior.rb

```diff
@@ -3,9 +3,9 @@ module ImportBehavior
     extend ActiveSupport::Concern
 
     def build_for_importer
-      build_metadata
-      raise CollectionsCreatedError unless collections_created?
       begin
+        build_metadata
```

Comment:
No, that was in ImportWorkJob where we only raise an error if the failure hasn't happened in the entry. Thinking about it, build metadata could be caused by a network error for OAI which we *would* want to raise again, but for CSV and RDF it would more likely be a data processing error which we wouldn't.

How about, I change the custom error class to something more generic and rescue/raise that in OAI but not in the other entries?

Reply:
yep